my-app/src/helpers/api.ts

```diff
@@ -0,0 +1,25 @@
+import axios from '../axios'
+import { Configuration, PostsApi } from '../swagger'
+import { useContext, useMemo } from 'react'
+import {LoginContext, LogoutContext} from '../pages/authentication/AuthContext'
+
+export function createConfiguration(authToken?: string) {
+  return new Configuration({
+    basePath: axios,
```

Comment:
problem 1: basePath is 'http://localhost:3222' , but you are passing object here
problem 2: your swagger.json does not correspond with routes on BE. That's why when I fixed basePath it was calling `/post` endpoint, but your backend was listening on `/posts` route